### ISSUE TITLE
 Invite or Add User input field of create team and discussion not behave properly.

### ISSUE THREAD

**Issue by anikdhabal** [RocketChat/Rocket.Chat#28919]
### Description:
When trying to invite or add multiple user in the input field of create team and discussion, the user suggestion list is automatically closed after select a user. 

### Steps to reproduce:

1. Click the create new button
2. team or discussion
3. try to add multiple user or invite member using the user list

### Expected behavior:

The list should behave like this:-
![Screenshot 2023-04-14 045519 (2)](https://user-images.githubusercontent.com/81948346/231905128-6bbfd57f-8b3e-407c-a977-ca080f8a7ddf.jpg)

### Actual behavior:

https://user-images.githubusercontent.com/81948346/231905685-f2ae8076-54ef-42d3-8493-80ca13e86442.mp4

### Server Setup Information:

-Version of Rocket.Chat Server: 6.0.0-develop
-Operating System: windows 10
-Deployment Method: Gitpod
-Number of Running Instances:
-DB Replicaset Oplog: Enabled
-NodeJS Version: 14.21.2 - x64
-MongoDB Version:5.0.5

**Comment by anikdhabal** [comment]
@hugocostadev @debdutdeb can you check it sir?

**Comment by debdutdeb** [comment]
Yeah I agree with you. I'd prefer the behaviour you're highlighting. But we need to quickly run this by design team for confirmation. I'll let Hugo handle that :)

**Comment by Dass404** [comment]
[click here](https://www.mysourcebook.com/2023/04/invite-or-add-user-input-field-of-create-team-and-discussion-not-behave-properly-in-rocket-chat.html) to read this article. May be this article help you solve this issue.

**Comment by hugocostadev** [comment]
One more behavior to check: https://github.com/RocketChat/Rocket.Chat/issues/28936

**Comment by Dass7** [comment]
May be this is article help you to solve this issue [click here to read](https://www.mysourcebook.com/2023/04/invite-or-add-user-input-field-of-create-team-and-discussion-not-behave-properly-in-rocket-chat.html)

**Comment by dougfabris** [comment]
Hi there, thanks for the contribution! 🚀 💯 

Sorry, this error does not happen on the latest and maintained releases.

[Rocket.Chat](https://rocket.chat/)  provides support for, at least, 3 months for each release and 6 months for the latest releases of each major version

I would suggest you update your Rocket.Chat version to the latest stable version, which you can find in [Releases Page](https://github.com/RocketChat/Rocket.Chat/releases) and, check it [HERE](https://docs.rocket.chat/resources/get-support/enterprise-support#rocket.chat-versions) the table with versions and End of Life dates. 

---

**Questions? Help needed? Feature Requests?** 

- Join our [Open Server](https://open.rocket.chat/) in the #support channel and feel free to raise a question
- Join our [Community Forum](https://forums.rocket.chat/) and search/create a post there
